spec/features/edit_draft_work_spec.rb

```diff
@@ -20,7 +20,7 @@
       visit dashboard_path
 
       within('#deposits-in-progress') do
-        click_link work.title
+        click_link "Edit #{work.title}"
       end
```

Comment:
tests updated as a result of the change for #943 (title link no longer goes to edit page)